streams/src/test/java/org/apache/kafka/streams/kstream/internals/KTableFilterTest.java

```diff
@@ -44,13 +51,32 @@
     final private Serde<Integer> intSerde = Serdes.Integer();
     final private Serde<String> stringSerde = Serdes.String();
     private final Consumed<String, Integer> consumed = Consumed.with(stringSerde, intSerde);
+    private final ConsumerRecordFactory<String, Integer> recordFactory = new ConsumerRecordFactory<>(new StringSerializer(), new IntegerSerializer());
+    private TopologyTestDriver driver;
+    private Properties props;
+
     @Rule
-    public final KStreamTestDriver driver = new KStreamTestDriver();
-    private File stateDir = null;
+    //TODO: remove this driver when we can pass processor context to the getters using the TopologyTestDriver
```

Comment:
Can you provide some details for this issue?

Reply:
`KTableValueGetter` must be inited with the processor context, and I didn't find any easy way to do it, so I just skipped it and work on the remaining classes, and left this to workout at the end of the refactor. If anyone has some tips on how to do it the are very welcome.